src/mscorlib/Tools/BclRewriter/BclRewriter.targets

```diff
@@ -5,30 +5,32 @@
   <!-- ********************************************************************************************* -->
 
   <PropertyGroup>
-    <BclRewriterCommand Condition="'$(BclRewriterCommand)'==''">$(CoreCLRToolsDir)\BclRewriter.exe</BclRewriterCommand>
+    <IntermediateOutputRootPath Condition="'$(IntermediateOutputRootPath)' == ''">$(IntermediateOutputPath)</IntermediateOutputRootPath>
     <BclRewriterModelFile>$(MscorlibDir)model.xml</BclRewriterModelFile>
     <BclRewriterWorkDir>$(IntermediateOutputPath)\BclRewriter</BclRewriterWorkDir>
     <BclRewriterSymbolOutput>$(IntermediateOutputPath)\BclRewriter\$(TargetName).pdb</BclRewriterSymbolOutput>
     <BclRewriterOutput>$(IntermediateOutputPath)\BclRewriter\$(TargetName)$(TargetExt)</BclRewriterOutput>
+    <BuildToolsSemaphore>$(IntermediateOutputPath)\BclRewriter\semaphore</BuildToolsSemaphore>
```

Comment:
Looks like coreclr repo doesn't have this yet. Perhaps you can mimic it by using setting BuildToolsSemaphore to $(ToolsDir)Microsoft.DotNet.Build.Tasks.dll